src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const rightRobot = {};
+  const valueArr = Object.values(robot);
+  const sortArr = valueArr.sort();
+
```

Comment:
you can rewrite method `sort` like this: `sort((a, b) => a - b)`. Its better practice, because method sort can work incorrect
It sort array only by the first number, for example [1, 200, 40, 5].  It is better to specify the sort order directly. That's what Denis said!)